internal/terraform/rootmodule/walker.go

```diff
@@ -105,6 +111,12 @@ func (w *Walker) IsWalking() bool {
 
 func (w *Walker) walk(ctx context.Context, rootPath string, wf WalkFunc) error {
 	defer w.Stop()
+
+	excludePathMap := make(map[string]bool)
+	for _, path := range w.excludeModulePaths {
+		excludePathMap[path] = true
+	}
```

Comment:
Totally nitpick - we could be forming the data into the map even earlier - in the setter, instead of storing the data in "inappropriate form" and then converting them when we need them. I can move it in a separate commit though.